### DISCUSSION TITLE
scroll event w/ useEffect not rendering

### DISCUSSION THREAD

**Discussion by themyrtlecoder** [vercel/next.js#16703] [Help]
# Bug report

## Describe the bug

I am using Next.js 9.5.2. I have tried to incorporate a useEffect that console logs window.scrollY or window.pageYOffset on a scroll event. While the code works with Next.js 9.4.4, it will not work properly with the most recent version of Next.js 9.5.2.

I have tried a "click" event listener in place of "scroll", which works...very strange functionality. I have successfully created scroll functionality from scratch with different versions of Next.js previously and feel like I am missing something very small - please help!

## To Reproduce

the relevant code is posted below
_**please note - I have tested this same code block in a Next.js 9.4.4 application and it works perfectly fine**_: 

import Head from 'next/head';
import Layout from '../components/layout';
import styles from '../styles/index.module.scss';
import React, { useEffect, useState } from 'react';
import { Section } from '../components/transitions';

const Index = () => {
  useEffect (()=> {
    document.addEventListener('scroll', () => {
      console.log('position', window.scrollY)}); /* **_or_** console.log('offset', window.pageYOffset)*/
  },[])
  return (
    <Layout>
      ...//includes content that extends beyond the viewport
    </Layout>
  )
}
export default Index;

## Expected behavior

The code above should render the accurate window.scrollY or window.pageYOffset on every scroll "event".

## System information

- OS: macOS
- Version of Next.js: 9.5.2
- Version of Node.js: 13.5.0

**Comment by jamesmosier** [comment]
Hi @aturtle35. I ran this code with Next 9.5.2 and it worked as expected

<img width="703" alt="Screen Shot 2020-08-30 at 9 51 25 PM" src="https://user-images.githubusercontent.com/2854919/91675630-0b998700-eb0b-11ea-967c-f8a538065c62.png">

Do you have a repository that you could share demonstrating this issue?

**Reply by hegdeMadan** [reply]
I also encountered the same issue
